client/src/components/App.js

```diff
@@ -60,39 +60,61 @@ export default class App extends React.Component {
       timedelay: '',
       refreshDisabled: true,
       delayDisabled: false,
-      minimise: false,
+      minimise: window.navigator.userAgent.toLowerCase().includes('mobi')
```

Comment:
I love this, I have no idea what it is though. Might be worth including a comment just explaining it?